includes/class-post-types.php

```diff
@@ -345,6 +347,65 @@ public static function register_outbox_post_type() {
 		);
 	}
 
+	/**
+	 * Register the Object post type.
+	 */
+	public static function register_object_post_type() {
```

Comment:
Should this be renamed, too? And the comment? 
`register_post_post_type()` is probably not great. Maybe `register_ap_post_post_type()`? We could also leave it